mithril-aggregator/src/message_adapters/from_register_signer.rs

```diff
@@ -14,7 +14,9 @@ impl TryFromMessageAdapter<RegisterSignerMessage, Signer> for FromRegisterSigner
             party_id: register_signer_message.party_id,
             verification_key: register_signer_message.verification_key.try_into()?,
             verification_key_signature: register_signer_message.verification_key_signature,
-            operational_certificate: register_signer_message.operational_certificate,
+            operational_certificate: register_signer_message
+                .operational_certificate
+                .and_then(|o| o.try_into().ok()),
```

Comment:
Using .ok() hide eventual errors, you should propagate the error instead using a ?. Sadly you can't use ? inside closures so you will need to convert this to a match.